### PR TITLE
Format code

### PR DESCRIPTION
**Pull request by jkortega9008**
This is just for testing purpose

### CODE REVIEW

**Comment by andresmoschini** [comment]
You have a really long commit message

image

You could rebase interactively and change the commit messages by a shorter one or you could edit [.gitlint](https://github.com/FromDoppler/doppler-contact-policies-api/blob/main/.gitlint) to allow larger messages.

**Comment by andresmoschini** [comment]
I think that it is not a good idea making PRs against INT

**Comment by andresmoschini** [comment]
I think that it is not a good idea to make PRs against `INT` branch

image

File: Doppler.ContactPolicies.Business.Logic/Class1.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace Doppler.Contact.Policies.Business.Logic
+namespace Doppler.ContactPolicies.Business.Logic
```
Comment (ms-darianbenito) [inline Doppler.ContactPolicies.Business.Logic/Class1.cs:3]: What do you think about using this PR to rename the business logic layer too:

```suggestion
namespace Doppler.ContactPolicies.BusinessLogic
```

**Comment by jkortega9008** [comment]
Olvidé poner el comentario aquí, no quería que revisaran este PR, es que estaba probando el code format en mi PC, para saber si pasando lgunos comandos no fallaba en el CI. Pero mi intención no era mezclar a INT

File: Doppler.ContactPolicies.Business.Logic/Class1.cs (hunk 1 above)
Comment (jkortega9008) [reply Doppler.ContactPolicies.Business.Logic/Class1.cs:3]: I will in another PR, i dont want to mess with filenames and so on

**Pull request closed without merging**